test/helpers/utils.go

```diff
@@ -518,6 +518,10 @@ func GKENativeRoutingCIDR() string {
 	return os.Getenv("NATIVE_CIDR")
 }
 
+func AKSNativeRoutingCIDR() string {
```

Comment:
Nit: Seems like this does the same thing as `GKENativeRoutingCIDR()`, maybe consolidate these functions into `NativeRoutingCIDR()` instead?

Reply:
I thought of doing so. I am not fully versed with the impact it might have on changing the name `GKENativeRoutingCIDR()` to `NativeRoutingCIDR()`. There is only use one case for that function. Not sure if that'll break something for GKE test cases.

Reply:
Both of them rely on the `NATIVE_CIDR` so there's no special logic or unexpected impact that I can see.

Reply:
It's configuring the same option, so as long as it's reading the same environment variable I think this should be OK.